### PR TITLE
[10.0][MIG] website_portal_purchase

### PR DESCRIPTION
**Pull request by Tardo**
Migrated 'website_portal_purchase' to v10.0

New features in this PR:

    Now the menu mark as 'active' the selected item.

Depends:

    [server-tools]: 'auth_supplier' -> OCA/server-tools#793

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please preserve commit history following technical method explained in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0

File: website_portal_purchase/__manifest__.py [hunk 1]
```diff
@@ -7,11 +7,11 @@
     'name': 'Website Portal for Purchases',
     'category': 'Website',
     'summary': "Add purchase orders and quotation in the frontend portal",
-    'version': '9.0.1.0.0',
+    'version': '10.0.1.0.0',
```
Comment (tarteo) [inline website_portal_purchase/__manifest__.py]: Can you up the version number for the new feature.

File: website_portal_purchase/__manifest__.py [hunk 2]
```diff
@@ -27,5 +27,5 @@
               'Odoo Community Association (OCA)',
     'website': 'http://www.tecnativa.com',
```
Comment (tarteo) [inline website_portal_purchase/__manifest__.py]: Please change this to the repository url.

**Review by tarteo — CHANGES_REQUESTED** [review]

**Comment by rafaelbn** [comment]
Hello @Tardo 

Please preserve commit history following technical method explained in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0

Are you going to continue with this PR?

Thanks a lot!

**Comment by Tardo** [comment]
Yes i can finish this. This week i will send the changes.

**Review by tarteo — APPROVED** [review]
LGTM

**Pull request closed without merging**